setup.sh

```diff
@@ -18,6 +18,8 @@ if [[ "$unamestr" == 'Darwin' ]]; then
 fi
 echo '[INSTALL] Installing Requirements'
 pip install -r requirements.txt
+echo '[INSTALL] Updating APKiD rules'
+bash scripts/update_apkid.sh
```

Comment:
My concern is this script should be fail safe. if it failed is should not affect the rest of the setup process.

Reply:
I'm on it

Reply:
we can also remove it from install process so user launch it if they want :)

Reply:
but with last change it shouldn't fail